c7n/resources/awslambda.py

```diff
@@ -79,6 +80,11 @@ def get_resources(self, ids, cache=True, augment=False):
         return super(AWSLambda, self).get_resources(ids, cache, augment)
 
 
+@AWSLambda.filter_registry.register('cel')
+class CELFilter(BaseCELFilter):
+    RelatedIdsExpression = ""
```

Comment:
what's related id expressions got to do with cel base filter?

Reply:
My mistake, I'm adding CEL support for RelatedResourceFilter concurrently in a different branch. Deleting this from Lambdas in this PR for now.